common/utils/ServiceBrokerClient.js

```diff
@@ -72,6 +73,31 @@ class ServiceBrokerClient extends HttpClient {
       }, 200)
       .then(res => res.body);
   }
+
+  updateServiceInstance(options) {
```

Comment:
The intention of this PR was to decouple update from CF and if i understood it right allow for bind during update. If the /v2/service_instances patch api still returns back 202, then during normal updates bind wont go through because CF itself would error out. So the scope of this PR is to directly hit broker end point for update?

Reply:
only for schedule update.